Source/JavaScriptCore/llint/InPlaceInterpreter.asm

```diff
@@ -980,6344 +642,766 @@ else
     break
 end
 
-if WEBASSEMBLY and (ARM64 or ARM64E or X86_64)
-# Put all instructions after this `if`, or 32 bit will fail to build.
+# Value-representation-specific code.
+if JSVALUE64 and (ARM64 or ARM64E or X86_64)
+    include InPlaceInterpreter64
+elsif ARMv7
+    include InPlaceInterpreter32_64
+else
+# For unimplemented architectures: make sure that the assertions can still find the labels
```

Comment:
Not sure off hand. This is how it was when I found it, so I didn't dare to change it. I guess at this point the only unimplemented architecture is x86?

FWIW I appreciated the unimplemented "template" being there when starting the armv7 port because you can copy paste it and start filling holes.

Reply:
@kmiller68 Ok if we do this potentially in a separate PR? Or is this a blocker for you?

Reply:
No that’s fine